static/sass/_snapcraft_metrics.scss

```diff
@@ -43,4 +46,17 @@
       pointer-events: none;
     }
   }
+
+  .snapcraft-metrics__active-devices {
```

Comment:
Is this used anywhere?

Reply:
Ah, now I've made the names go to the left of the | it's not needed, removing thanks